packages/dashboard/src/pages/Wallets/components/TokenTable/index.tsx

```diff
@@ -0,0 +1,168 @@
+import { Dispatch, memo, SetStateAction, useState } from 'react'
+import {
+    Table,
+    TableContainer,
+    TableHead,
+    TableRow,
+    TableCell,
+    Box,
+    makeStyles,
+    TableBody,
+    Pagination,
+    PaginationItem,
+} from '@material-ui/core'
+import { MaskColorVar } from '@dimensiondev/maskbook-theme'
+import { useDashboardI18N } from '../../../../locales'
+import { EmptyPlaceholder } from '../EmptyPlaceholder'
+import { LoadingPlaceholder } from '../LoadingPlacholder'
+import { TokenTableRow } from '../TokenTableRow'
+import { useAssets, useERC20TokensPaged } from '../../hooks'
+import { formatBalance } from '@dimensiondev/maskbook-shared'
+import BigNumber from 'bignumber.js'
+import { ceil } from 'lodash-es'
+import type { Asset } from '../../types'
+
+const useStyles = makeStyles((theme) => ({
+    container: {
+        height: 'calc(100% - 58px)',
+        display: 'flex',
+        flexDirection: 'column',
+        maxHeight: 'calc(100% - 58px)',
+    },
+    header: {
+        color: MaskColorVar.normalText,
+        fontWeight: theme.typography.fontWeightRegular,
+        padding: '24px 28px',
+        backgroundColor: MaskColorVar.primaryBackground,
+    },
+    footer: {
+        flex: 1,
+        display: 'flex',
+        justifyContent: 'center',
+        alignItems: 'center',
+    },
+    paginationItem: {
+        borderRadius: 4,
+        border: `1px solid ${MaskColorVar.lineLight}`,
+        color: MaskColorVar.textPrimary,
+        '&.Mui-selected': {
```

Comment:
Avoid this

Reply:
https://next.material-ui.com/customization/how-to-customize/#pseudo-classes

https://github.com/mui-org/material-ui/issues/26565

Reply:
Okay... But can't we use `<PaginationItem {...item} classes={{ root: classes.paginationItem, selected: classes.pagniationItemSelected }} />` or things like that?

Reply:
I tried it and it didn't work